emu_mps/mps_backend.py

```diff
@@ -55,7 +58,15 @@ def run(self) -> Results:
         impl = create_impl(self._sequence, self._config)
         impl.init()  # This is separate from the constructor for testing purposes.
 
-        return self._run(impl)
+        results = self._run(impl)
+        if not self._config.optimize_qubit_ordering:
```

Comment:
I would extract this to separate function, something like
```
results = self._run(impl)

return reorder(results, config)
```

Reply:
now its `impl.permute_results(results)`